### PR TITLE
Add clink 0.4.5 (windows only)

### PR DESCRIPTION
**Pull request by tadeu**

### CODE REVIEW

**Comment by msarahan** [comment]
Sorry you closed this - did we not get to it fast enough?  Please ping me on Windows issues like this.  I use Cmder, which uses Clink and find it helpful.  I'd be happy to include it here.

File: clink/activate_clink.bat [hunk 1]
```diff
@@ -0,0 +1,2 @@
+@echo off
+call clink inject -q --profile "%LOCALAPPDATA%"\clink
```
Comment (msarahan) [inline clink/activate_clink.bat]: Is there a way to make sure that clink is removed cleanly when this package is uninstalled?

File: clink/bld.bat [hunk 2]
```diff
@@ -0,0 +1,6 @@
+@mkdir %SCRIPTS% 2> NUL
+@for %%f in (clink.bat clink.lua clink_inputrc_base clink_x64.exe clink_dll_x64.dll clink_x86.exe clink_dll_x86.dll) do (
+    move %SRC_DIR%\%%f %SCRIPTS%
+    @if errorlevel 1 exit 1
+)
+xcopy %RECIPE_DIR%\activate_clink.bat %PREFIX%\etc\conda\activate.d\
```
Comment (msarahan) [inline clink/bld.bat]: newline here?  I think it runs fine without it, but I like to keep them present.

File: clink/bld.bat [hunk 3]
```diff
@@ -0,0 +1,6 @@
+@mkdir %SCRIPTS% 2> NUL
+@for %%f in (clink.bat clink.lua clink_inputrc_base clink_x64.exe clink_dll_x64.dll clink_x86.exe clink_dll_x86.dll) do (
```
Comment (msarahan) [inline clink/bld.bat]: You don't copy the license.  Does that make the package non-GPL compliant?

**Comment by tadeu** [comment]
@msarahan I closed it because I didn't think it was useful: at least for me, I abandoned `clink` in favor of using `cmder` (which already combines `ConEmu`, `clink` and other useful things).

**Pull request closed without merging**